### PR TITLE
tcno-acc-switcher: Add version 3.0.02

### PR DESCRIPTION
**Pull request by ghost-420**

### CODE REVIEW

**Comment by ghost-420** [comment]
globals.json is able to be created, but "persist" doesn't keep the file as its orginial name if the extension is ',json', instead of something such as '.ini'. Why could that be?

File: bucket/tcno-acc-switcher.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            "url": "https://github.com/TcNobo/TcNo-Acc-Switcher/releases/download/3.0.02/TcNo.Account.Switcher.x64.zip",
+            "hash": "42d17824569d311451e78cf5f12333f1c6af2dc9080b68c5226c741e4e2aa1e4",
+            "extract_dir": "TcNo.Account.Switcher.x64",
+            "pre_install": [
+                "# Create placeholder files",
+                "function CreateFile([String[]] $file) {",
+                "    foreach ($f in $file) {",
+                "        if (! (Test-Path \"$dir\\$f\")) { New-Item \"$dir\\$f\" -ItemType 'File' | Out-Null }",
+                "    }",
+                "}",
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:19]: - Why this file is created only in 64bit??
- Full function for creating one file is completely nonsense...

File: bucket/tcno-acc-switcher.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{
+    "version": "3.0.02",
+    "description": "A super fast account switcher for Steam.",
+    "homepage": "https://github.com/TcNobo/TcNo-Acc-Switcher",
+    "license": "GPL-3.0-only,LGPL-2.1-only,...",
+    "notes": ".NET Framework 4.8 is required.",
+
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:7]: ....

File: bucket/tcno-acc-switcher.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        "globals.json",
+        "Steam\\SteamSettings.json",
+        "Steam\\SteamVACCache.json",
+        "Steam\\Tray_Users.json",
+        "Steam\\Errors",
+        "Steam\\images"
+    ],
+	"post_install": [
+		"Remove-Item \"$dir\\`$*\", \"$dir\\TcNo Acc Switcher Updater.exe\" -Recurse"
+    ],
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:58]: Move this into pre_install

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/tcno-acc-switcher.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            "TcNo Account Switcher"
+        ],
+        [
+            "Steam\\TcNo Acc Switcher SteamTray.exe",
+            "[Tray] TcNo Account Switcher - Steam"
+        ],
+        [
+            "Steam\\TcNo Account Switcher Steam.exe",
+            "TcNo Account Switcher - Steam"
+        ]
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:46]: Installer does not create any of these shortcuts

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/tcno-acc-switcher.json (hunk 1 above)
Comment (ghost-420) [reply bucket/tcno-acc-switcher.json:19]: Why are you reviewing this when it's clearly incomplete?

Comment (Ash258) [reply bucket/tcno-acc-switcher.json:19]: Why do you open PR when it is clearly incomplete?

File: bucket/tcno-acc-switcher.json (hunk 2 above)
Comment (ghost-420) [reply bucket/tcno-acc-switcher.json]: What needs to be changed there?

File: bucket/tcno-acc-switcher.json (hunk 1 above)
Comment (ghost-420) [reply bucket/tcno-acc-switcher.json]: I was being lazy and hoped someone would do the work for me.

**Comment by ghost-420** [comment]
All the .json files need 
```
{}
```
as their file contents to not crash this program.
I'll go do that now.

File: bucket/tcno-acc-switcher.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        "    foreach ($f in $file) {",
+        "        if (! (Test-Path \"$dir\\$f\")) { New-Item \"$dir\\$f\" -ItemType 'File' | Out-Null }",
+        "    }",
+        "}",
+        "CreateFile 'globals.json'",
+        "CreateFile 'Steam\\SteamSettings.json'",
+        "CreateFile 'Steam\\SteamVACCache.json'",
+        "CreateFile 'Steam\\Tray_Users.json'",
+        "Remove-Item \"$dir\\`$*\", \"$dir\\TcNo Acc Switcher Updater.exe\" -Recurse"
+    ],
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:46]:
```suggestion
```

File: bucket/tcno-acc-switcher.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            "url": "https://github.com/TcNobo/TcNo-Acc-Switcher/releases/download/3.0.02/TcNo.Account.Switcher.x64.zip",
+            "hash": "42d17824569d311451e78cf5f12333f1c6af2dc9080b68c5226c741e4e2aa1e4",
+            "extract_dir": "TcNo.Account.Switcher.x64"
+        },
+        "32bit": {
+            "url": "https://github.com/TcNobo/TcNo-Acc-Switcher/releases/download/3.0.02/TcNo.Account.Switcher.x32.zip",
+            "hash": "16c62e3392821f082993a1f5c6bf1c21550351f6f72f7ee6a85d33d84a48f486",
+            "extract_dir": "TcNo.Account.Switcher.x32"
+        }
+    },
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:18]:
```suggestion
    },
    "pre_install": [
        "'globals.json', 'Steam\\SteamSettings.json', 'Steam\\SteamVACCache.json', 'Steam\\Tray_Users.json' | ForEach-Object {",
        "    $cont = if ($_ -like '*Tray_Users*') { '[]' } else { '{}' }",
        "    if (!(Test-Path \"$persist_dir\\$_\")) { Set-Content \"$dir\\$_\" $cont -Encoding Ascii | Out-Null }",
        "}",
        "Remove-Item \"$dir\\TcNo Acc Switcher Updater.exe\""
    ],
```

File: bucket/tcno-acc-switcher.json [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    "bin": "TcNo Account Switcher.exe",
+    "shortcuts": [
+        [
+            "TcNo Account Switcher.exe",
+            "TcNo Account Switcher"
+        ]
+    ],
+    "persist": [
+        "globals.json",
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:27]:
```suggestion
        "Errors",
        "globals.json",
```

File: bucket/tcno-acc-switcher.json [hunk 8]
```diff
@@ -0,0 +1,58 @@
+{
+    "version": "3.0.02",
+    "description": "A super fast account switcher for Steam.",
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:3]:
```suggestion
    "description": "A super fast account switcher for Steam",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/tcno-acc-switcher.json [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+        "    if (! (Test-Path \"$dir\\$file\")) {",
+        "        New-Item \"$dir\\$file\" -ItemType File -Value $content | Out-Null",
+        "    }",
+        "}",
+        "CreateFile 'globals.json' '{}'",
+        "CreateFile 'Steam\\SteamSettings.json' '{}'",
+        "CreateFile 'Steam\\SteamVACCache.json' '{}'",
+        "CreateFile 'Steam\\Tray_Users.json' '{}'",
+        "Remove-Item \"$dir\\`$*\", \"$dir\\TcNo Acc Switcher Updater.exe\" -Recurse"
+    ],
```
Comment (Ash258) [inline bucket/tcno-acc-switcher.json:46]:
```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**